File: multi_lang/python_templates/model_template.py
"""Template: SQLAlchemy ORM model (raw). Placeholders are substituted by PythonModelGenerator."""
from sqlalchemy import Column, Float, Integer, String
from sqlalchemy.orm import declarative_base

Base = declarative_base()


class __MODEL_NAME__(Base):
    """ORM model for the __MODULE_NAME__ domain entity."""

    __tablename__ = "__ROUTE__"

    id = Column(Integer, primary_key=True, index=True)
    name = Column(String(255), nullable=False)
    price = Column(Float, nullable=False)

File: multi_lang/go_templates/model_template.go
// Template: GORM model (raw). Placeholders substituted by GoModelGenerator.
package main

import "gorm.io/gorm"

type __MODEL_UPPER__ struct {
	ID    uint    `gorm:"primaryKey"`
	Name  string  `gorm:"size:255;not null"`
	Price float64 `gorm:"not null"`
}

func __MODEL_UPPER__Table(db *gorm.DB) *gorm.DB {
	return db.Model(&__MODEL_UPPER__{})
}